docs/seqcol.md

```diff
@@ -385,6 +386,58 @@ ga4gh:
     - sequences
 ```
 
+##### Example of a service-info response
+
+```json
+{
+  "id": "org.ga4gh.refget.seqcol",
+  "name": "Sequence Collection server",
+  "type": {
+    "group": "org.ga4gh",
+    "artifact": "refget.seqcol",
+    "version": "1.0.0"
+  },
+  "description": "Collection of sequences from digests.",
+  "organization": {
+    "name": "My organization",
+    "url": "https://example.com"
+  },
+  "contactUrl": "mailto:support@example.com",
+  "documentationUrl": "https://docs.myservice.example.com",
+  "createdAt": "2024-01-01T11:00:00Z",
+  "updatedAt": "2024-01-01T11:00:00Z",
+  "environment": "prod",
+  "version": "1.4.1",
+  "seqcol": {
+    "schema": {
+      "description": "A collection of biological sequences.",
+      "type": "object",
+      "$id": "https://example.com/sequence_collection",
```

Comment:
Shouldn't this really be a functioning URL where we share our base schema? I believe we discussed this some time ago, and I suspect we put this on the TODO list for things to do after v1.0. Which is fine for me. However, perhaps some comment should be added about this?

Reply:
Update: It is already commented in the spec:

> The base seqcol schema will be made available as the spec is finalized.

I believe we are finalizing the spec right now, so perhaps this is the time to make the schema available as a separate JSON document?